playwright/README.md

```diff
@@ -0,0 +1,75 @@
+# Playwright Lunch 'n' Learn
+
+## What is PlayWright?
+
+[Playwright](https://playwright.dev/) is a end-to-end testing library. It can run tests in multiple browser environments to ensure proper operation of our application.
+
+## Overview
+
+Playwright will connect with the web dev server so we can connect test out the latest Web UI code with an existing deephaven-core server.
```

Comment:
The wording seems off here: connect with the web dev server so we can **connect test out the latest**...
Delete the word `connect` or append the word `and` after it?